### DISCUSSION TITLE
Modals Development Abandoned?

### DISCUSSION THREAD

**Discussion by ZK-273** [discordjs/discord.js#10214] [Ideas]
For a while now, I've been monitoring any improvements to Modals for Discord.js.
Nothing really breaking and notable has changed or has been added recently.

Basic features like:
Booleans
Menu Select Dropdowns
Radial Buttons
Sliders
Datepicker
Inline field verification
Buttons on modal ..
... etc.

Now some of the above may not be technically possible and even limited by Discord themselves.....  To which I'm not sure.
However, it seems we need some if not most of the above functionalities to make modals useful for Discord.js.

I'd like to make a member application form in Discord that isn't just a few freeform text fields.
Seems there's really no way to make anything worthwhile the effort due to the limitations and lack of functionality of Modals...

**Comment by vladfrangu** [comment]
All of the things you've mentioned are not doable because discord doesn't support those types. You'll have to suggest those to Discord before we can implement them!
